frontend/redux/sagas.js

```diff
@@ -36,11 +38,22 @@ export function* requestWorkload() {
   }
 }
 
+export function* completeWorkload() {
+  try {
+    const workloadToSend = yield select(selectWorkload);
+    const workloadPostResponse = yield call(postWorkload, workloadToSend);
+    yield put(workloadPostResponse);
+  } catch (e) {
+    console.log('I FAILED TO POST ANNOTATIONS', e);
```

Comment:
Reasoning: We want to be able to update "loading" status and render errors to the client (not in console.log)